bin/hr/hr.go

```diff
@@ -220,9 +221,14 @@ func (c *converter) transform(r io.Reader) {
 				continue
 			}
 		}
+
+		var (
```

Comment:
Please collapse in one line `var foo type`

Reply:
ok